templates/typescript_gapic/package.json

```diff
@@ -10,6 +10,23 @@
     "build/src",
     "build/protos"
   ],
+  "keywords": [
+    "google apis client",
+    "google api client",
+    "google apis",
+    "google api",
+    "google",
+    "google cloud platform",
+    "google cloud",
+    "cloud",
+    "google {{ api.naming.productName.toCamelCase() }}",
+    "{{ api.naming.productName.toCamelCase() }}",
+{%- set lastItem = api.uniqkeywords.pop() -%}
```

Comment:
This is actually not needed. Remove the comma in line 23, and in your template, put the comma _before_ the `{{ words }}`, not after. It will work. Don't care about formatting since we always post-format JSONs after generation.

Reply:
Thank you! I was struggling with the format and line splitter.